samples/02_power_users_developers/introduction_to_hub_initiatives_and_sites.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>Only the site item has been cloned in the new organization as this is an Enterprise organization. The script will automatically determine whether to give you an initiative or a site for your particular Hub.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>